datalad/local/tests/test_copy_file.py

```diff
@@ -0,0 +1,339 @@
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Test copy_file command"""
+
+
+from os.path import (
```

Comment:
BTW, in many places we already moved to use `from datalad.support import path as op` which is our "centralized" shims for `os.path` and then `op.join` which isn't too long ;)  ATM it seems there is still lots of `opj` (~700) but `op.join` is catching up (~500).  Such centralization of imports / shims could allow to tune / fix/ workaround in case of need.

Reply:
I hope that in most places we have moved away from using either model, and use `from datalad.utils import Path`.

Reply:
yes, I should use it more often; but comment was for non `pathlib` ways, which is what was used here.